repos/system_upgrade/el7toel8/actors/commonleappdracutmodules/libraries/modscan.py

```diff
@@ -46,7 +47,10 @@ def _create_dracut_modules():
 
 
 def _create_initram_packages():
-    return RequiredUpgradeInitramPackages(packages=_REQUIRED_PACKAGES)
+    required_pkgs = _REQUIRED_PACKAGES
+    if architecture.matches_architecture(architecture.ARCH_X86_64):
+        required_pkgs.append('biosdevname')
```

Comment:
Didn't you want to remove the biosdevname from the _REQUIRED_PACKAGES?

Reply:
@bocekm Oh man... Yes! I will update tests first to catch that and then I will fix it.